src/relay/pass/let_list.h

```diff
@@ -0,0 +1,75 @@
+/*!
+ *  Copyright (c) 2018 by Contributors
+ * \file let_list.h
+ * \brief Let List Let you Let stuff yo.
+ */
+#ifndef TVM_RELAY_PASS_LET_LIST_H_
+#define TVM_RELAY_PASS_LET_LIST_H_
+
+#include <utility>
+#include <vector>
+#include <tuple>
+#include "tvm/relay/type.h"
+#include "incomplete_type.h"
+
+namespace tvm {
+namespace relay {
+
+struct PartialVar {
+  Var v;
+  explicit PartialVar(const Var & v) : v(v) { }
```

Comment:
Why is PartialVar necessary, can we use Var?

Reply:
it is there to signal 'I am incomplete, you have to eventually letlist::plug me'
it is removed as I am only taking the plug with me right now.